components/molecules/MobileComponent.vue

```diff
@@ -0,0 +1,234 @@
+                                lazy
+                                placeholder="000000"
+                            />
+                        </b-form-group>
+                    </b-col>
+                </b-row>
+                <div class="text-center mt-3 text-grey font-italic">
+                    <small v-if="props.verifyPhone">
+                        Once this step is complete, we will send you another SMS that should be used
+                        for login on the next screen.
+                    </small>
+                    <small v-else>
+                        Please contact our
+                        <a
+                            class="text-primary text-decoration-none"
+                            href="https://support.weavr.io/support/login"
+                            target="_blank"
+                            >customer support</a
+                        >
+                        if you have difficulties signing in.
+                    </small>
+                </div>
+                <LoaderButton :is-loading="isLoading" class="mt-4 text-center mb-0" text="verify" />
+            </form>
+            <div v-if="props.verifyPhone">
+                <b-alert
+                    :show="dismissCountDown"
+                    class="text-center mt-4 mb-0 text-muted small"
+                    variant="white"
+                    @dismiss-count-down="countDownChanged"
+                >
+                    {{ dismissCountDown }}
+                    <slot name="countdown"></slot>
+                </b-alert>
+                <div v-if="!dismissCountDown" class="mt-4 text-center">
+                    <small class="text-grey">
+                        Didn’t receive a code?
+                        <b-link class="text-decoration-underline text-grey" @click="resendSms"
+                            >Send again
+                        </b-link>
+                        .
+                    </small>
+                </div>
+            </div>
+        </b-card>
+    </div>
+</template>
+
+<script lang="ts">
+import {
+    defineComponent,
+    ref,
+    computed,
+    reactive,
+    useRoute,
+    useRouter,
+    useFetch,
+} from '@nuxtjs/composition-api'
+import { useStores } from '~/composables/useStores'
+import { useBase } from '~/composables/useBase'
+import { SCAOtpChannelEnum } from '~/plugins/weavr-multi/api/models/authentication/additional-factors/enums/SCAOtpChannelEnum'
+import {
+    AuthVerifyEnrol,
+    AuthVerifyEnrolSchema,
+    INITIAL_AUTH_VERIFY_REQUEST,
+} from '~/plugins/weavr-multi/api/models/authentication/additional-factors'
+import ErrorAlert from '~/components/molecules/ErrorAlert.vue'
+import LoaderButton from '~/components/atoms/LoaderButton.vue'
+import useZodValidation from '~/composables/useZodValidation'
+
+export default defineComponent({
+    components: {
+        ErrorAlert,
+        LoaderButton,
+    },
+    layout: 'auth',
```

Comment:
should this contain a layout definition ?

Reply:
it was present before